style.css

```diff
@@ -365,4 +366,21 @@ body .is-layout-flow > * {
 
 body .is-layout-constrained > * + * {
 	margin-block-start: 0px;
+}
+
+/* Design for search box in 404 page template sidebar */
+.course-search-sidebar button {
```

Comment:
We already have a search bar done on the search template. Can we reuse it?

Reply:
Thanks for the suggestion @gabrielcaires <3 I couldn't reuse the style in the search template the search button there is added using CSS and SVG file, the style added for the search button on search template depends on the search button being empty, but I used the built in block settings for the icon. So I added the style. Is there any special reason why we didn't use the icon setting in the search template? I used it this way -

https://user-images.githubusercontent.com/6820724/199707736-d060a23c-f2d4-4134-b1b9-2987a5958b75.mov